src/components/views/voip/CallDuration.tsx

```diff
@@ -32,20 +33,22 @@ export const CallDuration: FC<CallDurationProps> = memo(({ delta }) => {
     return <div className="mx_CallDuration">{formatPreciseDuration(delta)}</div>;
 });
 
-interface GroupCallDurationProps {
-    groupCall: GroupCall;
+interface SessionDurationProps {
+    session: MatrixRTCSession | undefined;
 }
 
 /**
- * A call duration counter that automatically counts up, given a live GroupCall
+ * A call duration counter that automatically counts up, given a matrixRTC session
  * object.
  */
-export const GroupCallDuration: FC<GroupCallDurationProps> = ({ groupCall }) => {
+export const SessionDuration: FC<SessionDurationProps> = ({ session }) => {
     const [now, setNow] = useState(() => Date.now());
+
     useEffect(() => {
         const timer = window.setInterval(() => setNow(Date.now()), 1000);
         return () => clearInterval(timer);
     }, []);
 
-    return groupCall.creationTs === null ? null : <CallDuration delta={now - groupCall.creationTs} />;
+    const createdTs = session?.getOldestMembership()?.createdTs();
```

Comment:
Might be worth commenting the shortcut here, ie. that this is the oldest current membership, so if the oldest member leaves, the duration will change.